src/bp/ui-shared/src/telemetry/index.ts

```diff
@@ -0,0 +1,47 @@
+import axios, { AxiosInstance } from 'axios'
+import { TelemetryEvent } from 'common/telemetry'
+import _ from 'lodash'
+import ms from 'ms'
+
+export const sendTelemetry = async (events: TelemetryEvent[]) => {
+  try {
+    await axios.post(
+      window.TELEMETRY_URL,
+      events.map(e => ({ ...e, source: 'client' }))
+    )
+    return true
+  } catch (err) {
+    console.error('Could not send the telemetry packages to the storage server', err)
+    return false
+  }
+}
+
+export const startFallback = async (api: AxiosInstance) => {
+  await sendSavedEvents(api)
+  setInterval(async () => await sendSavedEvents(api), ms('30m'))
+}
+
+const sendSavedEvents = async (api: AxiosInstance) => {
+  const events = await getSavedEvents(api)
+  if (!events.length) {
+    return // break recursivity
+  }
+  const success = await sendTelemetry(events)
+  await sendFeedback(api, events, success)
+
+  await sendSavedEvents(api)
+}
+
+const getSavedEvents = async (api: AxiosInstance): Promise<TelemetryEvent[]> => {
+  try {
+    const { data } = await api.get(`/telemetry/events`)
+    return _.compact(data)
```

Comment:
How about fixing the underlying issue instead ?? 

![image](https://user-images.githubusercontent.com/42552874/89308691-8d97ac80-d640-11ea-8441-44af8bff0da0.png)

Reply:
Fixed in 53445b1!